doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -307,6 +308,31 @@ response from the NCP acknowledging the command (with `CMD_VALUE_IS`).
 Once that acknowledgement is received the host may enter the low-power
 state.
 
+### PROP 4104: PROP_UNSOL_UPDATE_FILTER {#prop-unsol-update-filter}
+
+* Required only if `CAP_UNSOL_UPDATE_FILTER` is set.
+* Type: Read-Write
+* Packed-Encoding: `A(I)`
+* Default value: Empty.
+
+Contains a list of properties which are excluded from generating
+unsolicited value updates. This property **MUST** be empty after reset.
+
+Hosts **SHOULD NOT** add properties to this list which are not
+present in `PROP_UNSOL_UPDATE_LIST`. If such properties are added,
+the NCP **MUST** ignore the unsupported properties.
```

Comment:
Would it be more appropriate for the NCP to simply fail the request in its entirety, rather than taking some of the properties?  I think bug discovery might be easier in that case.